assets/scss/core/_base.scss

```diff
@@ -78,15 +76,120 @@ textarea {
     border: 1px solid $dp-color-yellow;
   }
 }
-/* validate form*/
 
+.checkmark {
+  position: absolute;
+  left: 0;
+  height: 16px;
+  width: 16px;
+  border: 1px solid $dp-color-grey;
+  border-radius: $dp-border-radius;
+  background-color: $dp-color-white;
+  pointer-events: none;
+
+  &:after {
+    content: '';
+    position: absolute;
+    left: 5px;
+    top: 1px;
+    width: 5px;
+    height: 10px;
+    border: solid $dp-color-white;
+    border-width: 0 3px 3px 0;
+    transform: rotate(45deg);
+  }
+}
+
+/* validate form*/
 .field-group {
   .field-item {
     position: relative;
     display: inline-block;
     text-align: left;
+    float: left;
     width: 100%;
     margin-bottom: $dp-spaces--lv5;
+
+    &__checkbox {
+      cursor: pointer;
+
+      label {
+        font-weight: normal;
+        vertical-align: middle;
+        margin-left: 25px;
+      }
+    }/* Field checkbox */
+
+    &--50 {
+      width: 50%;
+      /* Left Column */
+      &:nth-of-type(2n+1) {
+        padding-right: 20px;
```

Comment:
Are you still using arbitrary spacings? or it is missing take advantage of Gustavo variables?

Reply:
This was already corrected in the next commit 👍